### PR TITLE
Get version from Version.scala

### PR DESCRIPTION
**Pull request by lewfish**
Previously, I was trying to get SBT to get the version number from the `TRAVIS_TAG` environment variable, but it should actually get it from Version.scala.

See #26 for discussion

### CODE REVIEW

File: project/Version.scala [hunk 1]
```diff
@@ -4,7 +4,8 @@ object Version {
   def either(environmentVariable: String, default: String): String =
     Properties.envOrElse(environmentVariable, default)
 
-  val floodmodel = either("TRAVIS_TAG", "SNAPSHOT")
+  val version = "0.1.3"
+  val floodmodel = version + either("FLOODMODEL_VERSION_SUFFIX", "-SNAPSHOT")
```
Comment (rajadain) [inline project/Version.scala]: Where will `FLOODMODEL_VERSION_SUFFIX` be set?

File: README.md [hunk 2]
```diff
@@ -172,7 +172,7 @@ To build the server JAR, execute the following:
 $ ./sbt "project server" assembly
 ```
 
-which should result in `server/target/scala-2.10/usaceflood-server-assembly-SNAPSHOT.jar`.
+which should result in `server/target/scala-2.10/usaceflood-server-assembly-<version>-SNAPSHOT.jar`.
```
Comment (rajadain) [inline README.md]: Consider also updating the JAR name in other places, like https://github.com/azavea/usace-flood-geoprocessing/pull/27/files#diff-04c6e90faac2675aa89e2176d2eec7d8R193

File: README.md [hunk 3]
```diff
@@ -211,6 +211,7 @@ Deployments to GitHub Releases are handled via [Travis-CI](https://travis-ci.org
 ``` bash
 $ git flow release start 0.1.0
 $ vim CHANGELOG.md
+$ vim project/Version.scala
```
Comment (rajadain) [inline README.md]: Consider adding explanation for what is to be changed in these files.

File: project/Version.scala (hunk 1 above)
Comment (lewfish) [inline project/Version.scala]: It's supposed to be an optional environment variable. Currently, it's not being used. I'm not sure why it was included in the original commit https://github.com/azavea/usace-flood-geoprocessing/blob/bf1724b934b3b17391d23c7c9704b84f321aa431/project/Version.scala

Comment (rajadain) [inline project/Version.scala]: Oh so you're just reverting this. Got it.

**Comment by lewfish** [comment]
Updated according to feedback.

**Comment by rajadain** [comment]
+1 code and documentation looks good. Travis succeeded, as did a local build:

```bash
$ ./sbt "project server" assembly
...
[info] Packaging server/target/scala-2.10/usaceflood-server-assembly-0.1.3-SNAPSHOT.jar ...
[info] Done packaging.
[success] Total time: 23 s, completed Jan 21, 2016 10:29:31 AM
```

**Pull request merged**